CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# Contributing to Crane
+
+Welcome to Crane! This document is a guideline about how to contribute to Crane.
+
+If you find something incorrect or missing, please leave comments / suggestions.
+
+## Become a contributor
+
+You can contribute to Crane in several ways. Here are some examples:
+
+* Contribute to the Crane codebase.
+* Contribute to the [Crane docs](https://github.com/gocrane/crane).
+* Report bugs.
+* Suggest enhancements.
+* Write technical documentation and blog posts, for users and contributors.
+* Organize meetups and user groups in your local area.
+* Help others by answering questions about Crane.
+
+For more ways to contribute, check out the [Open Source Guides](https://opensource.guide/how-to-contribute/).
+
+### Contribute to the Crane codebase
+
+We encourage new comers actively join in Crane projects. The following paragraph introduce how to contribute in Crane way.
+
+#### Begin your contribute
+
+If you find a typo in document, find a bug in code, or want new features, or want to give suggestions, you can [open an issue on GitHub](https://github.com/gocrane/crane/issues/new) to report it.
```

Comment:
how about change `Contribute to the Crane codebase` to `Develop Guide`. 
You can rename `workflow.MD` for detail